docs/chain_info.md

```diff
@@ -0,0 +1,43 @@
+# ChainInfo
+
+## Intro
+
+This correct definition of these objects is instrumental to the correct behaviour of our services and web app.
+These objects contain information that coordinates the inter-service communication as well as template URLs that allows the frontend to render links to block explorers correctly. These features will be explained in this doc.
```

Comment:
I find these two paragraphs a little bit confusing. I do understand them because of the context that I have with the service. Also `ChainInfo` doesn't really exist in the Safe Config Service – there the model instance is called `Chain`

What do you think about the following?

> The Chain object represents metadata specific to a chain. Its metadata is crucial to not only have a the correct information displayed to the users (chain name, chain logo, native currency, etc...) but also to communicate with the correct instance of the Safe Transaction Service (`Transaction Service URI` and `VPC Transaction Service URI`)

> The Chain object also contains data which is used specifically in the context of the Safe Web App and the Safe Mobile apps. This data is used mostly for rendering purposes like currency logo, background colours, etc. This data is required but affect only the visual components of the client application.

Reply:
Nice suggestion! I will add it, I would also try to briefly mention URL templates too, as it has its own section later in the document. I want to hint in the intro what topics will be covered in more detail later on :) Thanks for the suggestion!